tvtk/pyface/ui/qt4/decorated_scene.py

```diff
@@ -158,6 +158,7 @@ def _save_snapshot(self):
                             "Geomview", "VRML", "Wavefront", "Open Inventor",
                             "Povray", "X3D"]
 
+            wildcard = ""
```

Comment:
Because it is triggered only when you click on "save" and I am not sure the best way to test that.  Do you have any suggestions of doing this easily without popping up a UI?